themes/default/content/docs/intro/languages/java.md

```diff
@@ -10,7 +10,9 @@ menu:
 
 <img src="/logos/tech/java.svg" align="right" width="90" style="padding:8px; margin-top: -64px">
 
-Pulumi supports writing your infrastructure as code using the Java language. Java 11 or later is required. We support running Maven Pulumi programs, Gradle Pulumi programs, and Pulumi programs packaged as JAR files.
+Pulumi supports writing your infrastructure as code using the Java language running on any [Supported version](https://www.oracle.com/java/technologies/java-se-support-roadmap.html) after Java 11.
+
+We support running Maven Pulumi programs, Gradle Pulumi programs, and Pulumi programs packaged as JAR files.
```

Comment:
Great start. FWIW Maven and Gradle have documented lower bounds https://github.com/pulumi/pulumi-java#requirements  - but I don't think we figured out the LTS strategy yet. We can refine this later. LGTM.